src/test/java/RenamePlaylistTest.java

```diff
@@ -0,0 +1,25 @@
+import org.testng.annotations.Test;
+
+public class RenamePlaylistTest extends BaseTest {
+
+
+    @Test
+
+    public void renamePlaylist() {
+
+
+        // GIVEN user is on the login page ("https://bbb.testpro.io/");
+        driver.get(url);
+        // AND user puts the correct login and password info
```

Comment:
**Please, remove code redundancies**

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )


![image](https://user-images.githubusercontent.com/104761482/224248183-d11184fa-6ce4-42d0-8e7c-70bb9e14e979.png)